src/Transition.js

```diff
@@ -394,7 +394,7 @@ Transition.propTypes = {
    *     (see
    *     [test/CSSTransition-test.js](https://github.com/reactjs/react-transition-group/blob/13435f897b3ab71f6e19d724f145596f5910581c/test/CSSTransition-test.js#L362-L437)).
    */
-  nodeRef: PropTypes.shape({ current: PropTypes.instanceOf(Element) }),
+  nodeRef: PropTypes.shape({ current: PropTypes.instanceOf(typeof Element === 'undefined' ? function() {} : Element) }),
```

Comment:
Better not to rely on Element at all to support different windows http://tobyho.com/2011/01/28/checking-types-in-javascript/.

Reply:
```suggestion
  nodeRef: PropTypes.shape({ 
    current: typeof Element === 'undefined' 
      ? PropTypes.any
      : PropTypes.instanceOf(Element) 
   }),
```